doc/user/output.xml

```diff
@@ -135,8 +117,16 @@ Type: 'scons program' to build the production program,
 
     <para>
 
-    When used with &AddOption; Help("text", append=False) will clobber any help output associated with AddOption().
-    To preserve the help output from AddOption(), set append=True.
+    Calling <literal>Help("text")</literal> will clobber
+    any help output associated with &f-link-AddOption; calls.
+    To preserve the help output from &AddOption;,
+    add the <literal>append=True</literal> keyword argument.
```

Comment:
from this wording I though append and local_only were AddOption() arguments, and not Help() arguments.
Can you rephrase (I guess same comment for CHANGES and RELEASE blurbs)

Reply:
yeah, my bad, didn't sufficiently rewrite existing text.